src/QueryM/DatumCacheWsp.purs

```diff
@@ -12,35 +12,41 @@ module QueryM.DatumCacheWsp
   , requestMethodName
   ) where
 
+import Prelude
+
 import Aeson
   ( Aeson
+  , (.:)
+  , caseAesonArray
+  , caseAesonObject
   , decodeAeson
   , getNestedAeson
   , toStringifiedNumbersJson
   )
-import Control.Alt (map, (<$), (<$>), (<|>))
-import Control.Bind ((=<<), bind)
-import Control.Category ((<<<))
+import Control.Alt ((<|>))
+import Control.Applicative (unless)
+import Control.Monad (unlessM)
 import Data.Argonaut
   ( Json
   , JsonDecodeError
-      ( UnexpectedValue
-      , AtIndex
+      ( AtIndex
       , Named
+      , TypeMismatch
+      , UnexpectedValue
       )
   , decodeJson
   , encodeJson
   , jsonNull
   , stringify
   )
 import Data.Bifunctor (lmap)
-import Data.Either (Either(Left, Right), note)
+import Data.Either (Either(Left, Right), fromRight, note)
 import Data.Eq (class Eq)
-import Data.Function ((>>>), const, ($))
+import Data.Function (const)
 import Data.Maybe (Maybe(Just, Nothing), maybe)
-import Data.Monoid ((<>))
 import Data.Newtype (unwrap)
 import Data.Show (class Show)
```

Comment:
Redundant imports. Just going to put these in one post (some of these are surprisingly part of `prelude` apparently)
```
import Control.Monad (unlessM)
right, fromRight
import Data.Function (const)
import Data.Show (class Show)
import Data.Eq (class Eq)
import Data.Unit (Unit, unit)
```

Reply:
Ah thanks. Previously there were no imports from `Prelude` in this module (e.g. `$` was imported from `Data.Function`). I wanted to change that since it can be annoying to keep track of imports at such a granular level. Must have missed those